apps/emqx_authz/test/emqx_authz_api_sources_SUITE.erl

```diff
@@ -29,17 +29,17 @@
 
 -define(SOURCE1, #{<<"type">> => <<"http">>,
                    <<"enable">> => true,
-                   <<"url">> => <<"https://fake.com:443/">>,
+                   <<"base_url">> => <<"https://fake.com:443/">>,
+                   <<"path">> => <<"foo">>,
+                   <<"query">> => <<"a=b">>,
                    <<"headers">> => #{},
                    <<"method">> => <<"get">>,
                    <<"request_timeout">> => <<"5s">>
                   }).
 -define(SOURCE2, #{<<"type">> => <<"mongodb">>,
                    <<"enable">> => true,
                    <<"mongo_type">> => <<"sharded">>,
-                   <<"servers">> => [<<"127.0.0.1:27017">>,
-                                     <<"192.168.0.1:27017">>
-                                    ],
+                   <<"servers">> => <<"127.0.0.1:27017,192.168.0.1:27017">>,
```

Comment:
The type before was a list.  Does this option support both comma separated servers and a list?  Maybe it's worth to test both?

Reply:
These tests had been commented out, so they were a bit outdated.

Type for `servers` is binary now: 
https://github.com/emqx/emqx/blob/master/apps/emqx_connector/src/emqx_connector_mongo.erl#L253